File: src/main/java/com/zynchub/digital/hubservice/app/mapper/ConnectorsMapper.java
package com.zynchub.digital.hubservice.app.mapper;

import com.zynchub.digital.hubservice.app.model.ConnectorsInfo;
import com.zynchub.digital.hubservice.app.dto.ConnectorsResponseDto;

public class ConnectorsMapper {

    public ConnectorsResponseDto mapConnectorsData(ConnectorsInfo connectorsInfo){
        return new ConnectorsResponseDto(connectorsInfo.getStatus(), connectorsInfo.getMessage());
    }

}

File: internal/app/mapper/connectors_mapper.go
package mapper

import (
	"plexus-bff-service-go/internal/app/dto"
	"plexus-bff-service-go/internal/app/model"
)

type ConnectorsMapper struct{}

func NewConnectorsMapper() *ConnectorsMapper {
	return &ConnectorsMapper{}
}

func (m *ConnectorsMapper) Map(info model.ConnectorsInfo) dto.ConnectorsResponse {
	return dto.ConnectorsResponse{
		Status:  info.Status,
		Message: info.Message,
	}
}
